addons/crm/static/src/views/crm_kanban/crm_kanban_view.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="crm.KanbanView" t-inherit="web.KanbanView" t-inherit-mode="primary">
```

Comment:
This is quite a "meh" approach, can't we make it a bit less invasive and have less diff @jeh-odoo @frbin-odoo ? Purpose would be to tone down added JS when possible.